main.cpp

```diff
@@ -110,6 +115,13 @@ void display_next_combo_name(size_t tries, const std::vector<std::string>& combo
 
 int main()
 {
+	Env finder(std::filesystem::current_path());
+
+	PathSearch to_find;
+	to_find.push_back(std::filesystem::path("data/words.txt"));
```

Comment:
No need for explicit typename, `stdfs::path` has an implicit constructor from `char const*`.
```cpp
PathSearch to_find = "data/words.txt";
```

Reply:
I was getting errors using `{}` initialization specifically `vector out of range`

Reply:
Interesting; okay leave it then.